lib/ical/stringify.js

```diff
@@ -114,7 +114,7 @@ ICAL.stringify = (function() {
 
     var groupName = params.group;
     var line;
-    if (groupName && designSet.propertyGroups) {
+    if (designSet.propertyGroups && groupName) {
```

Comment:
This one's on me. I updated the two repos separately, and obviously missed this line.

Reply:
How is this a “spelling correction”?  In case both sides of `&&` are false, the former code will execute faster.